src/herbpy/tsr/fuze.py

```diff
@@ -0,0 +1,33 @@
+import numpy
+from prpy.tsr.tsrlibrary import TSRFactory
+from prpy.tsr.tsr import *
+
+@TSRFactory('herb', 'fuze_bottle', 'grasp')
+def fuze_grasp(robot, fuze, manip=None):
```

Comment:
I see, that makes sense. In that case, does the `TSRFactory` return a grasp with the end-effector directly adjacent to the bottle?

If so, it seems like `PushGrasp` should add the standoff distance to the beginning of the `TSRChain`. Otherwise, assuming a roughly symmetric distribution, there is a 50% chance that the wrist will hit the object we're trying to grasp during the `PlanToTSR` call.